docs/python_binding.rst

```diff
@@ -14,37 +14,39 @@ building
 Building the libtorrent python bindings will produce a shared library (DLL)
 which is a python module that can be imported in a python program.
 
-building using setup.py
------------------------
+building using boost build (windows)
+------------------------------------
 
-There is a ``setup.py`` shipped with libtorrent that can be used on windows.
-On windows the setup.py will invoke ``bjam`` and assume that you have boost
-sources at ``$BOOST_PATH``. The resulting executable is self-contained, it does
-not depend any boost or libtorrent dlls.
+Download and install `Visual C++ 2015 Build Tools <http://landinghub.visualstudio.com/visual-cpp-build-tools/>`_
 
-On other systems, the setup.py is generated by running
-``./configure --enable-python-binding``.
+Download `Boost libraries <http://www.boost.org/users/history/>`_ Extract it to c:\\Libraries\\boost_1_63_0 and create these environmental vars:
```

Comment:
are the double backslashes required by restructured text?

Reply:
nope, I changed all of them to forward slashes